policy/templates/releng/.github/workflows/release.yml.d/goreleaser.gotmpl

```diff
@@ -97,6 +97,19 @@
             *.txt.sig
             *.txt
 
+  {{- if and (eq .Name "tyk") (not (has "go1.19" .Branchvals.Features) ) }}
```

Comment:
This code is because of the plugin-compiler right? As long as the plugin-compiler depends on vendoring, this code is needed. I think maybe call the feature `plugin-compiler-vendoring` rather than have `(not (has "go1.19"` which looks fragile to me.